### ISSUE TITLE
Add CKAN support

### ISSUE THREAD

**Issue by slizzered** [charfa/XTLandertron#12]
[CKAN](https://github.com/KSP-CKAN/CKAN) is a package management system for KSP mods. It offers one-click install (including dependency management) and update of all mods.

[Adding support for a mod](https://github.com/KSP-CKAN/CKAN/wiki/Adding-a-mod-to-the-CKAN) is really easy :)

**Comment by charfa** [comment]
Yeah, I use CKAN myself and was thinking about it. But adding it to CKAN would mark my version as "The Official Landertron Release" and I wanted to hold on with that for a bit...
